server/controllers/ssotRetrievalController.js

```diff
@@ -178,3 +178,36 @@ exports.overwriteRobot = async (req, res) => {
     console.error(err);
   }
 };
+
+// DELETE /78d09f66d2ed466cf20b06f7
+exports.deleteRobot = async (req, res) => {
+  try {
+    res.set('Content-Type', 'application/json');
+    const { robotId } = req.params;
+    const usableRobotId = mongoose.Types.ObjectId(robotId);
+
+    const response = await mongoose
+      .model('SSoT')
+      .deleteOne({ _id: usableRobotId })
+      .exec();
+
+    await mongoose
+      .model('userAccessObject')
+      .deleteMany({ robotId: usableRobotId })
+      .exec();
+
+    await mongoose
+      .model('rpaAttributes')
+      .deleteMany({ robotId: usableRobotId })
+      .exec();
+
+    await mongoose
+      .model('parameter')
+      .deleteMany({ robotId: usableRobotId })
+      .exec();
+
```

Comment:
Please also delete all job objects that still are in the database. (It was the cause for a big bug we had last week)